op-challenger/game/fault/types/position.go

```diff
@@ -82,9 +82,13 @@ func (p Position) lshIndex(amount Depth) *big.Int {
 
 // TraceIndex calculates the what the index of the claim value would be inside the trace.
 // It is equivalent to going right until the final depth has been reached.
+// Note: this method will panic if maxDepth < p.depth
 func (p Position) TraceIndex(maxDepth Depth) *big.Int {
 	// When we go right, we do a shift left and set the bottom bit to be 1.
 	// To do this in a single step, do all the shifts at once & or in all 1s for the bottom bits.
+	if maxDepth < p.depth {
+		panic("maxDepth < p.depth")
```

Comment:
Perhaps `fmt.Sprintf` with the values that caused the panic itself

Reply:
Good idea! Done.